src/content/en/2019/compression.md

```diff
@@ -213,32 +220,33 @@ Additionally, the percentage of Brotli compression is higher for third party con
    <td><p style="text-align: right">0.01%</p></td>
   </tr>
   <tr>
-   <td>other / invalid
-   </td>
+   <td><em>Other / Invalid</em></td>
    <td><p style="text-align: right">0.01%</p></td>
    <td><p style="text-align: right">0.02%</p></td>
    <td><p style="text-align: right">0.01%</p></td>
    <td><p style="text-align: right">0.02%</p></td>
   </tr>
 </table>
 
-# Identifying Compression Opportunities
+## Identifying Compression Opportunities
 
-[Google’s Lighthouse](https://developers.google.com/web/tools/lighthouse) tool enables users to run a series of audits against web pages, and[ one of them](https://developers.google.com/web/tools/lighthouse/audits/text-compression) evaluates whether a site can benefit from additional text based compression. It does this by attempting to compress resources and evaluate whether an object’s size can be reduced by at least 10% and 1400 bytes. Depending on the score, you may see a compression recommendation in the results, with a list of specific resources that could be compressed.
+[Google’s Lighthouse](https://developers.google.com/web/tools/lighthouse) tool enables users to run a series of audits against web pages, and [one of them](https://developers.google.com/web/tools/lighthouse/audits/text-compression) evaluates whether a site can benefit from additional text based compression. It does this by attempting to compress resources and evaluate whether an object’s size can be reduced by at least 10% and 1400 bytes. Depending on the score, you may see a compression recommendation in the results, with a list of specific resources that could be compressed.
 
-![](../../../static/images/2019/15_Compression/ch15_fig8_lighthouse.jpg)
+![Lighthouse Compression Suggestions](../../../static/images/2019/15_Compression/ch15_fig8_lighthouse.jpg)
 
-Because the HTTP Archive runs Lighthouse audits for each mobile page, we can aggregate the scores across all sites to learn how much opportunity there is to compress more content. Overall, 62% of websites are passing this audit and almost 23% of websites have scored below a 40. This means that over 1.2 million websites could benefit from enabling additional text based compression.
+Because the [HTTP Archive runs Lighthouse audits](./methodology) for each mobile page, we can aggregate the scores across all sites to learn how much opportunity there is to compress more content. Overall, 62% of websites are passing this audit and almost 23% of websites have scored below a 40. This means that over 1.2 million websites could benefit from enabling additional text based compression.
```

Comment:
Deep-linking to the LH section
```suggestion
Because the [HTTP Archive runs Lighthouse audits](./methodology#lighthouse) for each mobile page, we can aggregate the scores across all sites to learn how much opportunity there is to compress more content. Overall, 62% of websites are passing this audit and almost 23% of websites have scored below a 40. This means that over 1.2 million websites could benefit from enabling additional text based compression.
```